spring-cloud-azure-samples/spring-cloud-stream-eventhub-kafka-sample/src/main/resources/application.properties

```diff
@@ -5,6 +5,9 @@ spring.cloud.azure.region=[region]
 spring.cloud.azure.eventhub.namespace=[eventhub-namespace]
 
 # For example here, the destination name of input and output should be the same.
+# Eventhub name must start with a letter or number, and can contain only letters, numbers, and the dash (-) character.
```

Comment:
Container part could be removed. As I remember, event hub name can only be lowercase, right?